public/js/controllers/address.js

```diff
@@ -83,11 +81,11 @@
             var _this = this
             $.getScript('/js/dygraphs.min.js', () => {
                 _this.typesGraphOptions = {
-                    labels: ['Date', 'RegularTx', 'Tickets', 'Votes', 'RevokeTx'],
-                    colors: ['#2971FF', '#41BF53', 'darkorange', '#FF0090'],
-                    ylabel: '# of Tx Types',
+                    labels: ['Date', 'SentTx (tx)', 'ReceivedTx (tx)', 'Tickets (sstx)', 'Votes (ssgen)', 'RevokedTx (ssrtx)'],
```

Comment:
I suggest the following wording:

'Sending (regular)', 'Receiving (regular)', 'Tickets', 'Votes', 'Revocations'

Reply:
ok. Noted